tests/comparison/test_EqualityChecker.py

```diff
@@ -19,7 +19,7 @@ def test_super_init_call(self, mocker):
         expected_call_args = {
             0: {
                 "args": (),
-                "kwargs": {"columns": ["a", "b"], "verbose": False, "copy": False},
+                "kwargs": {"columns": ["a", "b"], "verbose": False},
```

Comment:
P: Huge praise for going through and pulling out all of the `copy` args - it must have been a real chore. This keeps us in line with the pandas API, which will actively warn when people use `copy` from pandas >=3.0.